assets/js/crdt.js

```diff
@@ -365,13 +369,16 @@ class CRDT {
     }
 
     /**
-     * Delete the `character` received from the CRDT structure if present
+     * Delete the `character` received from the CRDT structure if present.
+     * Returns -1 if character is not deleted (i.e. when already locally deleted)
```

Comment:
when locally deleted or already deleted by someone else before whose operation reached first.
I guess the explanation is not needed.

Reply:
Right.